src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java

```diff
@@ -38,48 +51,71 @@ public FixedFieldCodesGroup getFixedFieldCodesGroupsByLeader(
       parameter.put("leader", leader);
       parameter.put("code", code);
       parameter.put("valueField", valueField);
-      injectDefaultValues(fixedFieldCodesGroup, storageService, parameter, fixedFieldCodesGroup.getHeaderTypeCode(), lang, configuration);
+      injectDefaultValues(fixedFieldCodesGroup, storageService, parameter, fixedFieldCodesGroup.getHeaderTypeCode(),
+          lang, configuration);
       return fixedFieldCodesGroup;
     }, tenant, okapiUrl, configurator);
   }
 
   @GetMapping("/fixed-fields-code-groups")
-  public FixedFieldCodesGroup getFixedFieldCodesGroups(
-    @RequestParam(required = false) final String leader,
-    @RequestParam final String code,
-    @RequestParam final int headerTypeCode,
-    @RequestParam final String lang,
-    @RequestParam(required = false) final String valueField,
-    @RequestHeader(OKAPI_TENANT_HEADER_NAME) final String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
+  public FixedFieldCodesGroup getFixedFieldCodesGroups(@RequestParam(required = false) final String leader,
+      @RequestParam final String code, @RequestParam final int headerTypeCode, @RequestParam final String lang,
```

Comment:
I personally find it much easier to read the old format w/ one annotation per line.

Reply:
I have changed it